site/pages/third-party/paymasters.md

```diff
@@ -33,3 +33,69 @@ const client = createSmartAccountClient({
   },
 });
 ```
+
+## ERC20 Gas Sponsorship
+
+We are working on building support for an ERC20 paymaster!
+
+In the meantime, you could use a third-party paymaster, such as [Stackup](https://docs.stackup.sh/docs/paymaster-api), to sponsor ERC-20 gas. Here's an example using Stackup with the Alchemy SDK:
+
+```ts
+import { createMultiOwnerModularAccountClient } from "@alchemy/aa-accounts";
+import {
+  alchemyFeeEstimator,
+  createAlchemyPublicRpcClient,
+} from "@alchemy/aa-alchemy";
+import { deepHexlify, resolveProperties } from "@alchemy/aa-core";
+import { createClient, http } from "viem";
+import { sepolia } from "viem/chains";
+
+const chain = sepolia;
+// [!code focus:99]
+const stackupClient = createClient({
+  transport: http("https://api.stackup.sh/v1/paymaster/STACKUP_API_KEY"), // TODO: Replace with your stackup API key here (https://docs.stackup.sh/docs/paymaster-api)
+});
+
+const alchemyRpcClient = createAlchemyPublicRpcClient({
+  connectionConfig: { rpcUrl: "ALCHEMY_RPC_URL" },
+  chain,
+});
+
+const alchemyClient = await createMultiOwnerModularAccountClient({
+  chain,
+  account: {
+    signer,
+  },
+  transport: http("ALCHEMY_RPC_URL"),
+  gasEstimator: async (userOp) => ({
+    ...userOp,
+    callGasLimit: "0x0",
+    preVerificationGas: "0x0",
+    verificationGasLimit: "0x0",
+  }), // Stackup gas estimation
```

Comment:
What does stackup gas estiamtion mean? Does the end user need to do anything or are these just dummy values?